pip-dep/requirements.txt

```diff
@@ -1,13 +1,13 @@
 Flask>=1.0.2
 flask_socketio>=3.3.2
-syft-proto>=0.1.0a1
 lz4>=2.1.6
 msgpack>=0.6.1
 numpy>=1.16.0
+phe>=1.4.0
+syft-proto>=0.1.0a1
```

Comment:
Not sure why syft-proto have to be in every requirements*.txt?
I thought requirements.txt is always installed and others are optional.
I.e. installing only requiements_dev.txt or only requiements_udacity.txt is not sufficient.